lib/exq_ui_web.ex

```diff
@@ -55,7 +55,9 @@ defmodule ExqUIWeb do
   defp view_helpers do
     quote do
       # Use all HTML functionality (forms, tags, etc)
-      use Phoenix.HTML
+      import Phoenix.HTML
+      import Phoenix.HTML.Form
+      use PhoenixHTMLHelpers
```

Comment:
This is to provide backwards compability with phoenix_html 3.3 through the helpers package
`{:phoenix_html_helpers, "~> 1.0"},` as the built in recommendation in upgrading